sfge/src/main/java/com/salesforce/config/UserFacingMessages.java

```diff
@@ -16,7 +16,7 @@ public static final class RuleDescriptions {
 
     public static final class RuleViolationTemplates {
         public static final String APEX_NULL_POINTER_EXCEPTION_RULE =
-                "Operation [%s] is likely to throw a NullPointerException";
+                "%s is likely to throw a NullPointerException";
```

Comment:
The display string cannot get the lhs and rhs values on violation message without many more low level changes. Based on priorities, I think modifying the text to fit the context is easier.